tests/TestCase/View/InertiaWebViewTest.php

```diff
@@ -38,4 +41,51 @@ public function testRendersComponentName()
 
         $this->assertStringContainsString('&quot;Users\/Index&quot', $result);
     }
+
+    public function createViewBuilderForPassThru()
+    {
+        $request = new ServerRequest();
+        $response = new Response();
+
+        $this->View = (new ViewBuilder())
+            ->setClassName('Inertia\View\InertiaWebView')
+            ->setOption('_nonInertiaProps', ['one', 'two', 'three'])
+            ->build([], $request, $response);
+    }
```

Comment:
`setUp` method is already creating the view builder class, why you need to do it again?

Reply:
Hi, I needed the ViewBuilder to set the `_nonInertiaProps` option as it doesn't effect the other tests I've moved it to `setUp`